twitter/TweetCell.m

```diff
@@ -21,4 +23,44 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated {
     // Configure the view for the selected state
 }
 
+- (IBAction)didTapFavorite:(id)sender {
+    // Update the local tweet model
+    if (self.tweet.favorited == NO) {
+        self.tweet.favorited = YES;
+        self.tweet.favoriteCount += 1;
+        // Send a POST request to the POST favorites/create endpoint
```

Comment:
Because you're using the API Manager this comment is a bit inaccurate.

You don't actually care how the action is happening underneath the hood. That's for the API manager to take care of!